README.rst

```diff
@@ -210,20 +210,24 @@ Please see our `contributing guide <https://github.com/mne-tools/mne-bids/blob/m
 Cite
 ----
 
-If you use ``mne-bids`` in your work, please cite one of the following papers,
-depending on which modality you used:
+If you use ``mne-bids`` in your work, please cite:
+
+.. code-block:: Text
+
+    Appelhoff, S., Sanderson, M., Brooks, T., Vliet, M., Quentin, R., Holdgraf, C., Chaumon, M., Mikulan, E., Tavabi, K., Höchenberger, R., Welke, D., Brunner, C., Rockhill, A., Larson, E., Gramfort, A. and Jas, M. (2019). MNE-BIDS: Organizing electrophysiological data into the BIDS format and facilitating their analysis. Journal of Open Source Software 4: (1896).
+
+and one of the following papers, depending on which modality you used:
 
 `MEG <http://doi.org/10.1038/sdata.2018.110>`_
 ##############################################
 
 .. code-block:: Text
 
-   Niso, G., Gorgolewski, K. J., Bock, E., Brooks, T. L., Flandin, G., Gramfort, A.,
-   Henson, R. N., Jas, M., Litvak, V., Moreau, J., Oostenveld, R., Schoffelen, J.,
-   Tadel, F., Wexler, J., Baillet, S. (2018). MEG-BIDS, the brain imaging data
-   structure extended to magnetoencephalography. Scientific Data, 5, 180110.
-   http://doi.org/10.1038/sdata.2018.110
-
+  Appelhoff, S., Sanderson, M., Brooks, T., Vliet, M., Quentin, R., Holdgraf, C.,
+  Chaumon, M., Mikulan, E., Tavabi, K., Höchenberger, R., Welke, D., Brunner, C.,
+  Rockhill, A., Larson, E., Gramfort, A. and Jas, M. (2019). MNE-BIDS: Organizing
+  electrophysiological data into the BIDS format and facilitating their analysis.
+  Journal of Open Source Software 4: (1896).
```

Comment:
It should be meg paper here

Reply:
brrr really sorry about that. Fixed it now.